docs/map-layer-configuration.md

```diff
@@ -1,65 +1,145 @@
 
 ## XYZ
 
-| Property           | Mandatory | Meaning | Example |
-|--------------------|:---------:|---------|---------|
-| type               | x | Indicator that the layer is a XYZ tiled image layer, always `XYZ` here  | `"type": "XYZ"` |
-| url                | x | The URL of the service providing the image tiles | `"url": "https://geodata.nationaalgeoregister.nl/tiles/service/wmts/brtachtergrondkaart/EPSG:28992/{z}/{x}/{y}.png"` |
-| tileGridRef        |   | Identifier of the tile grid to use for this layer (has to be defined in `tileGridDefs` | `"tileGridRef": "dutch_rd"` |
\ No newline at end of file
+| Property           | Meaning   | Example |
+|--------------------|:---------:|---------|
+| **type**           | Indicator that the layer is a XYZ tiled image layer, always `XYZ` here  | `"type": "XYZ"` |
+| **url**            | The URL of the service providing the image tiles | `"url": "https://geodata.nationaalgeoregister.nl/tiles/service/wmts/brtachtergrondkaart/EPSG:28992/{z}/{x}/{y}.png"` |
+| tileGridRef        | Identifier of the tile grid to use for this layer (has to be defined in `tileGridDefs` | `"tileGridRef": "dutch_rd"` |
+
+## Style for Vectorlayers
+
+Mandatory properties:
+- Points require **`radius`** and/or **`iconUrl`**.
+- Poygons require **`fillColor`**.
+- Lines require **`strokeColor`** or **`strokeWidth`**.
+
+| Property           | Meaning | Example |
+|--------------------|:-------:|---------|
+| strokeColor        | see [color](https://openlayers.org/en/latest/apidoc/module-ol_style_Stroke-Stroke.html) | `"strokeColor": "purple"` |
+| strokeWidth        | see [width](https://openlayers.org/en/latest/apidoc/module-ol_style_Stroke-Stroke.html) | `"strokeWidth": 2` |
+| fillColor          | Point and Polygon, see [color](https://openlayers.org/en/latest/apidoc/module-ol_style_Fill-Fill.html) | `"fillColor": "rgba(155,153,51,0.5)"` |
+| label              | see [label](map-layer-configuration?id=label) |  |
+| radius             | see [radius](https://openlayers.org/en/latest/apidoc/module-ol_style_Circle-CircleStyle.html) | `"radius": 4` |
+| scale              | Point only, see [scale](https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html) | `"scale": 4` |
+| iconUrl            | Point only, see [src](https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html) | `"iconUrl": "./static/icon/circle.svg"` |
+| iconAnchor         | Point only, see [anchor](https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html) | `"anchor": [0.5, 37]` |
+| iconAnchorXUnits   | Point only, see [anchorXUnits](https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html) | `"anchorXUnits": "fraction"` |
+| iconAnchorYUnits   | Point only, see [anchorYUnits](https://openlayers.org/en/latest/apidoc/module-ol_style_Icon-Icon.html) | `"anchorYUnits": "pixels"` |
+
+#### Label
```

Comment:
A hint that besides `attribute`, `outlineColor`, `outlineWidth`, `fillColor`, `minResolution`, `maxResolution` every config property of  `ol/style/Text` can be used would be good. Consider then if listing offsetX et al  can be removed.